ApolloTeleop.java

```diff
@@ -0,0 +1,226 @@
+ * OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+ * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+
+package org.firstinspires.ftc.teamcode;
+
+import com.qualcomm.robotcore.eventloop.opmode.Disabled;
+import com.qualcomm.robotcore.eventloop.opmode.LinearOpMode;
+import com.qualcomm.robotcore.eventloop.opmode.TeleOp;
+import com.qualcomm.robotcore.hardware.Servo;
+import com.qualcomm.robotcore.util.Range;
+
+import org.firstinspires.ftc.robotcontroller.external.samples.HardwarePushbot;
+
+/**
+ * This OpMode uses the common Pushbot hardware class to define the devices on the robot.
+ * All device access is managed through the HardwarePushbot class.
+ * The code is structured as a LinearOpMode
+ * <p>
+ * This particular OpMode executes a POV Game style Teleop for a PushBot
+ * In this mode the left stick moves the robot FWD and back, the Right stick turns left and right.
+ * It raises and lowers the claw using the Gampad Y and A buttons respectively.
+ * It also opens and closes the claws slowly using the left and right Bumper buttons.
+ * <p>
+ * Use Android Studios to Copy this Class, and Paste it into your team's code folder with a new name.
+ * Remove or comment out the @Disabled line to add this opmode to the Driver Station OpMode list
+ */
+
+@TeleOp(name = "Apollo Teleop", group = "Apollo")
+//@Disabled //TODO Enable this to show program in TeleOP list - lasim lev
+public class ApolloTeleop extends LinearOpMode {
+
+    /* Declare OpMode members. */
+    // could also use HardwarePushbotMatrix class.
+    HardwareApollo robot = new HardwareApollo();
+
+    public static final double MID_SERVO = 0.5;
+    public static final double liftSpeed = 0.5;
+
+    @Override
+    public void runOpMode() {
+
+        double clawDownPosition = 0.17;
+        double clawUpPosition = 0.17;
+       // double liftPosition = 0.64;
+        double left = 0;
+        double right = 0;
+        double Doobi = 0.2;
+        double Mcqueen = 1;
+        double bibix = 1;
+        Mcqueen = Math.min(Mcqueen, 1);
+        Mcqueen = Math.max(Mcqueen, 4);
+
+
+
+        /* Initialize the hardware variables.
+         * The init() method of the hardware class does all the work here
+         */
+        robot.init(hardwareMap);
+
+        // Send telemetry message to signify robot waiting;
+        telemetry.addData("Say", "Hello Driver");    //
+        telemetry.update();
+
+        //clawPosition = robot.clawLeft.getPosition();
+        //liftPosition = robot.liftLeft.getPosition();
+
+
+        // Wait for the game to start (driver presses PLAY)
+        waitForStart();
+
+        // run until the end of the match (driver presses STOP)
+        while (opModeIsActive()) {
+
+            if (gamepad1.x) {
+                bibix = 2;
+                idle();
+            }
+
+            if (gamepad1.b) {
+                bibix = 1;
+                idle();
+            }
+
+            if (bibix == 1) {
```

Comment:
rename bibix drive_stick_direction_forward
and set its type to be boolean.

Then you do:
if (drive_stick_direction_forward) {
...
} else {
...
}